5-infrastructure/modules/base-env/boa-sql.tf

```diff
@@ -61,17 +61,9 @@ data "google_compute_subnetwork" "subnet" {
   self_link = each.value
 }
 
-module "private_access" {
```

Comment:
lets open an issue so we dont loose track of this for 3-networks

Reply:
Issue created https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/issues/14